inst/qml/Prophet.qml

```diff
@@ -0,0 +1,598 @@
+import QtQuick 			2.8
+import QtQuick.Layouts 	1.3
+import JASP.Controls 	1.0
+import JASP.Widgets 	1.0
+import JASP				1.0
+
+Form
+{
+	VariablesForm
+	{
+		AvailableVariablesList	{ name: "allVariablesList" }
+		AssignedVariablesList	{ name: "dependent";		title: qsTr("Dependent Variable");	suggestedColumns: ["scale"]; singleVariable: true													}
+		AssignedVariablesList	{ name: "time";				title: qsTr("Time");				suggestedColumns: ["nominal"]; singleVariable: true													}
+		AssignedVariablesList	{ name: "changepoints";		title: qsTr("Changepoints");		suggestedColumns: ["nominal"]; singleVariable: true													}
+		AssignedVariablesList	{ name: "capacity";			title: qsTr("Carrying Capacity");	suggestedColumns: ["scale"]; singleVariable: true; id: cap; enabled: growth.value === "logistic"	}
+		AssignedVariablesList	{ name: "minimum";			title: qsTr("Saturating Minimum");	suggestedColumns: ["scale"]; singleVariable: true; id: floor; enabled: growth.value === "logistic"	}
+		AssignedVariablesList	{ name: "covariates";		title: qsTr("Covariates");			suggestedColumns: ["scale"]; id: covs																}
```

Comment:
Maybe you could increase the size of this box for covariates so that it is clear that you can have more of them

Reply:
Yes, I didn't know that was possible, but it's a good idea!